app/components/nav/index.js

```diff
@@ -155,7 +155,7 @@ class Nav extends Component {
                 position: fixed;
                 left: 0;
                 transform: translate(0%, -200%);
```

Comment:
original position could work with -100% so the delay is not visible when toggling

Reply:
done